Frontend/src/locales/en.json

```diff
@@ -76,9 +76,21 @@
   },
   "photos":  {
     "myPhotos": "My Photos",
-    "friendsPhotos": "Photos your friends took"
+    "descriptionMyPhotos": "Here you can view all the photos you've uploaded for the Photowalk challenges. Photos your Friends like the best have the most heart icons.",
```

Comment:
Here you can view all the photos you've uploaded for the Photowalk challenges. Photos your friends like a lot have the most heart icons.